app.yaml

```
@@ -3,9 +3,15 @@ runtime: go114
 main: ./server
 
 handlers:
-  - url: /
+  - url: /api/.*
+    script: auto
+    secure: always
+  - url: /l/.*
+    script: auto
+    secure: always
+  - url: /(.*/)*([^/]*\.(html|js))$
+    static_files: frontend/dist/\2
+    upload: frontend/dist/.*\.(html|js)$
+  - url: /.*
     static_files: frontend/dist/index.html
     upload: frontend/dist/index.html
```

```suggestion
  - url: /.*
    static_files: frontend/dist/index.html
    upload: frontend/dist/index.html
    secure: always
```

`secure: always` をつけた方が良い気がする。